guides/source/asset_pipeline.md

```diff
@@ -21,11 +21,11 @@ What is the Asset Pipeline?
 The asset pipeline provides a framework to concatenate and minify or compress
 JavaScript and CSS assets. It also adds the ability to write these assets in
 other languages and pre-processors such as CoffeeScript, Sass and ERB.
-It allows assets in your application to be automatically combined with assets 
+It allows assets in your application to be automatically combined with assets
```

Comment:
It's whitespace change and do we allow such changes when we are updating a file without having too much git churn.